### ISSUE TITLE
AxisValueFormatter class not found

### ISSUE THREAD

**Issue by DanChaltiel** [PhilJay/MPAndroidChart-Realm#10]
I'm trying to follow [the tutorial](https://github.com/PhilJay/MPAndroidChart-Realm/wiki/Realm.io-database-integration#plotting-realm-data), but Android Studio cannot import any class named AxisValueFormatter.

The guide stated to write this code : 

``` Java
AxisValueFormatter formatter = new AxisValueFormatter() {
      @Override
      public String getFormattedValue(float value, AxisBase axis) {
          return results.get((int) value).getPlayerName();
      }

      @Override
      public int getDecimalDigits() { return 0; }
  };
```

I tried to search for "AxisValueFormatter" in the [MPAndroidChart github](https://github.com/PhilJay/MPAndroidChart/find/v3.0.2) the [MPAndroidChart github](https://github.com/PhilJay/MPAndroidChart-Realm/find/master), and indeed I couldn't find this exact class (some classes are close though).

Is this a library bug, a guide error or a fault of mine ?

**Comment by subsub** [comment]
Yeah I don't find it either. And the tutorial doesn't event explain where to use that `formatter` anywhere in the code.

**Comment by hlen56** [comment]
Hi guys, i am having the same issue and wondering if you guys found any solution?
